src/webui/routes/register.jl

```diff
@@ -12,8 +12,16 @@ function register(r::HTTP.Request)
     form = parseform(String(r.body))
     package = get(form, "package", "")
     isempty(package) && return json(400; error="Package URL was not provided")
-    occursin("://", package) || (package = "https://$package")
-    match(r"https?://.*\..*/.*/.*", package) === nothing && return json(400; error="Package URL is invalid")
+    is_ssh = startswith(package, "git@") && occursin(":", package)
+    if !is_ssh && !occursin("://", package)
+         package = "https://$package"
+    end
+    if endswith(package, ".git")
+        package = package[1:end-length(".git")]
```

Comment:
Why was this necessary? Did you get an error when you tried registering or do you not use .git in your company?

If this is necessary then it needs to be handled here https://github.com/JuliaRegistries/RegistryTools.jl/blob/03096523fe864f86ee374019a2ebea461e47d9be/src/register.jl#L331 as well (i.e do the equality check without ".git") or else it will give an error for registrations done using the publicly deployed registrator.

Reply:
It was necessary because the GitHub API called from forge returned 404 to me when it contained .git.

Reply:
I just tested out this PR and it seems to be fine. Sorry for the noise.